sys/mips/beri/beri_mp.c

```diff
@@ -225,14 +225,12 @@ platform_init_ap(int cpuid)
 
 	KASSERT(cpuid < MAXCPU, ("%s: invalid CPU id %d", __func__, cpuid));
 
-#if 0
```

Comment:
I am not sure why the `#if 0` was added in https://github.com/CTSRD-CHERI/cheribsd/commit/70ec1dbf2642789478fd8b7f3920fdad100f7a07. LGTM but I would wait for @brooksdavis to review.

Reply:
If I had to guess it wasn't implemented on the SMP core at the time.